src/index.js

```diff
@@ -19,6 +19,7 @@ const app = express();
 
 // swagger config middlewares
 app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
+app.enable('trust proxy');
```

Comment:
Why this?

Reply:
`app.enable('trust proxy');` enables our application to use `https` . By default `req.protocols = http` which is not a protected protocol. So when that is used it enables our server to use secured protocol. Why this was applied was in the generation of verification link. When we are on our local environment it evaluates to false and true when in production. So i used it in the construction of my verification link. for local enviroment we have something like `http://localhost...` and for production we have `https://...` To see this in application make a post request from `https://firestar-backend-staging-pr-24.herokuapp.com/api/v1/users/email/test` and visit your mail inbox you would observe the link on that mail received has a secured protocol  `https`

Reply:
Ok good.